src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1426,7 +1426,19 @@ public Map<String, String> getSimpleStatesWithPort()
 
     public String getGossipInfo(boolean withPort)
     {
-        return withPort ? fdProxy.getAllEndpointStatesWithPort() : fdProxy.getAllEndpointStates();
+        return getGossipInfo(withPort, false);
+    }
+
+    public String getGossipInfo(boolean withPort, boolean resolveIp)
+    {
+        if (resolveIp)
```

Comment:
if `if / else` is so simple, we tend to leave parenthesis completely, do this:

````
if (resolveIp)
    return ...
else
    return ...
````